README.md

```diff
@@ -1,2 +1,135 @@
 # Pokedex
-A simple web app that shows details concerning different Pokémon. Built using HTML,CSS, and JavaScript
+A simple web app that shows details concerning different Amiibo. Built using HTML,CSS, and JavaScript. Users view a list of amiibo and have the possibility to like and comment on their favorite amiibo.
+
+## Video presentation
+* [Video](https://drive.google.com/file/d/1uWRGgGf2-dIB1MxPkByqx1PbaKrkF1-2/view?usp=sharing)
```

Comment:
- [ ] Kindly record an additional video highlighting a piece of code that you found interesting or are proud of making.